src/net/dev.rs

```diff
@@ -96,7 +96,7 @@ named!(interface_stats<DeviceStatus>,
 
 named!(interface_list< Vec<DeviceStatus> >,
     do_parse!(
-        interfaces: separated_list!(line_ending, interface_stats) >>
+        interfaces: separated_list_complete!(line_ending, interface_stats) >>
```

Comment:
I'm not sure what the significance of this change is, but it was failing to unwrap Incomplete results until I changed this.